pywhat/printer.py

```diff
@@ -98,4 +99,13 @@ def pretty_print(self, text: dict, text_input):
         """
 
     def print_json(self, text: dict):
-        return json.dumps(text, indent=4)
+        if text['Regexes'] == None:
+            self.console.print('{}')
+
+        else:
+            delete_values = ['Regex', 'plural_name', 'Tags', 'Rarity']
+            values = text['Regexes']['text'][0]['Regex Pattern']
```

Comment:
It does not work for files, can you fix this somehow?🙏

Reply:
Are there any other input types I should be testing? I only tested str arguments.

Reply:
Nope! Just files and strings for this :)

Reply:
+folders